meta-iotqa/conf/test/refkit-image-common-extensive.manifest

```diff
@@ -1 +1,3 @@
 # Extensive tests for common profile
+oeqa.runtime.bluetooth.comm_bt_command
```

Comment:
These two classes take less than a minute to complete. However, there are still more Bluetooth multi-node test cases left to enable requiring a second device for testing connection. All these times will start adding up when more test cases are enabled for this and other features and executed for each profile image if they are common between them.

Also, I didn't include the comm_bt_stability test cases, as they take about half an hour to complete considering they are stress tests. At this moment, I'm not sure if they could be useful in the future, but if they are not, it'd probably be a good idea to delete them.